tests/TODO.md

```diff
@@ -15,6 +15,45 @@ functional unit or feature. The format of list is as follows:
 
 ### TESTS
 
+#### Backend:
+
+* **Stat commit errors**. This test checks if backend's `stat_commit_errors` is
+  taken into account during `Backend` status recalculation (when errors occur,
+  backend must be turned read-only).
+  1. *First error*. This test case checks basic functionality. Backend is first
+  initialized and enabled, then the status is calculated. It must be `OK`.
+  The next `BackendStat` is coming with non-zero `stat_commit_errors`. After the
+  status recalculation it must become `RO`.
+  2. *Subsequent errors*. This case checks if backend remains in state `RO`
+  both when nothing changes and when new errors occur. Send two updates:
+  stat with the same number of commit errors and update with greater number
+  of errors.
+  3. *Restart*. This case checks if `Backend` turns into state `OK` after
+  a clean restart. Read-only backend is updated with a stat with more recent
+  `last_start` timestamp.
+  4. *Reset*. Check if backend becomes `OK` when a new number of commit errors
+  is less than previously accounted.
+* **Read-only flag**. This test checks if value of `status/read_only` is taken
+  into account. Backend must be turned into state `RO`.
+  1. *Flag set*. Check if state transition from `OK` to `RO` works fine. Update
```

Comment:
According to test description backend is in `RO` state. Does this checks transition from `OK` to `RO`?